projects/ad74413r/src/platform/stm32/parameters.c

```diff
@@ -47,8 +47,6 @@ struct stm32_spi_init_param ad74413r_spi_extra_ip  = {
 	.chip_select_port = SPI_CS_PORT,
 };
 
-#ifdef IIO_TRIGGER_EXAMPLE
```

Comment:
ignore any iio trigger modifications, they were made because of a previous design mistake in this project.